.circleci/config.yml

```diff
@@ -183,7 +183,7 @@ jobs:
             mkdir -p build/logs/tmp build/cov
             for f in $(find features -name '*.feature' -not -path 'features/main/exposed_state.feature' | circleci tests split --split-by=timings); do
               _f=${f//\//_}
-              FEATURE="${_f}" phpdbg -qrr vendor/bin/behat --profile=coverage --suite=default --tags=~@postgres --format=progress --out=std --format=junit --out=build/logs/tmp/"${_f}" "$f"
```

Comment:
Why is this change needed?

Reply:
(It's probably this change that decreases the coverage)

Reply:
The coverage was already decreasing before this change.
I just saw that it was useless as the tags are defined in the [behat.yml.dist](https://github.com/api-platform/core/blob/master/behat.yml.dist#L61)
that's basically what I did a while ago, see https://github.com/api-platform/core/commit/71e37cbd17b17f21d36c9e52a143ecef0db9f8f5

I can revert if you prefer